TEMPLATE-SPEC.md

```diff
@@ -9,69 +9,123 @@ Requires: spec-xyz | ip-xyz
 
 # Spec-<Number>: <Title>
 
-## Summary
+## Overview
 
-Background and short summary in layman terms.
+<!-- One page high-level overview; put details in the specification section and background in the previous section. Should be understandable by a new engineer or reader who isn't already working on the project. -->
 
-## Context
+Short summary in layman terms.
+
+### Challenge
+
+<!-- List the challenge(s) being solved by this widget. What is the problem? -->
+<!-- One page high-level overview; put details in the specification section and background in the previous section. Should be understandable by a new engineer or reader who isn't already working on the project. -->
+
+### Context / Background
```

Comment:
Let's decide if we want to call it Context or Background. Personally I prefer Context.

Reply:
Context works for me!